### ISSUE TITLE
HTML5 player only if 60fps available

### ISSUE THREAD

**Issue by yunasc** [YePpHa/YouTubeCenter#1598]
Hi!

I would like to make feature request for player type:
currently flash player is more cpu efficient on my machine, but i want to watch 60fps videos.

currently i am using User-Agent switcher in FF nightly, but i would like to have this kind of option in YTC.

**Comment by AwesomeDonkey** [comment]
I don't believe Flash supports 60fps YouTube videos, hence the HTML5 requirement.

**Comment by ErdeFB** [comment]
I'd have to agree with @AwesomeDonkey's view on this. I don't think that YouTube officially supports 60fps videos with the flash player any longer, but I could be wrong.

Also, it could be Firefox issue. Does this problem still persist in version 37 that just came out, @yunasc?

**Comment by Zarggg** [comment]
As far as I am aware, YouTube never supported 60fps videos in the Flash Player, officially or otherwise.

**Comment by ErdeFB** [comment]
My apologies, I think I had some kind of brain fart.

This might actually be somewhat decent feature, but doesn't sound like a much utilized one. However, if it is rather trivial to implement, I would give my +1 for this.

But as @Zarggg said, YouTube seems to be phasing out Flash player so I guess eventually there won't really be much of an choice.

**Comment by Zarggg** [comment]
The 60fps videos do have specific format codes, so it might be possible by checking the list of available formats and if any of those are present, use the HTML5 player. Otherwise, use the Flash player.

**Comment by yunasc** [comment]
Yes, I meant that there would be option like player type, with setting "Flash, but HTML5 for 60fps videos".
